javascript/javascript1/week2/fullname.js

```diff
@@ -1,6 +1,6 @@
 // first part
 function getFullname (firstname, surname) {
- return (`"${firstname} ${surname}"`)
+ return `"${firstname} ${surname}"`
```

Comment:
There are the old way of declaring a string:
```js
let a = "I am a string"
```
and the newer way with string interpolation:
```js
let a = `I am a string`
```
:point_up:  notice without `"`.

Both will print the same:
```js
> console.log(a)
I am a string
```


Sometimes you want to use quotes inside a string like so:
```js
let b = `I am reading a book with the title: "JavaScript for beginners", and I am half way through`
```

I'm getting the feeling you think it necessary to include `"` - which it is not :smile: 

I see you doing this several of your JavaScript files

Reply:
Thanks for comment. But I put double quote mark around sentences and phrases because I just thought they are quoted. :).
In the homework was like this: getFullname("Benjamin", "Hughes", false); // returns "Benjamin Hughes". So I returned them in the same way that homework asked.

Reply:
I think you are mixing up declaration and printing:
```js
// here I'm declaring a string, the qoute is necessary for the computer to understand the value is a string
let name = "Hughes"

// here I'm printing the name (which is a string)
console.log(name)
// which will print the name WITHOUT qoutes
// > Hughes


// now I'm redeclaring name with string interpolation
name = `"Hughes"`

// and when I print it it WITH qoutes becuase the character " is part of the string content:
// > "Hughes"
```

you need to play with this in the console yourself so you notice how the computer behaves.